components/02-elements/search/search.hbs

```diff
@@ -0,0 +1,42 @@
+<div class="search-block">
+    <div class="search-block__toggle-btn">
```

Comment:
not sure about the design but I already made a compact form for newsletter, it might be helpful ##30674